src/tests/base.purchases_test.ts

```diff
@@ -0,0 +1,26 @@
+import { Purchases } from "../main";
+import { setupServer } from "msw/node";
+import { APIGetRequest, getRequestHandlers } from "./test-responses";
+import { afterAll, beforeAll, beforeEach } from "vitest";
+
+export const testApiKey = "rcb_test_api_key";
+export const testUserId = "someAppUserId";
+
+const server = setupServer(...getRequestHandlers());
+
+beforeAll(() => {
```

Comment:
I refactored these to another file, to avoid repetition and having all tests in the same file.